src/main/java/spoon/reflect/declaration/CtParameter.java

```diff
@@ -60,6 +60,12 @@ public interface CtParameter<T> extends CtVariable<T>, CtShadowable {
 	@UnsettableProperty
 	<C extends CtVariable<T>> C setDefaultExpression(CtExpression<T> assignedExpression);
 
+	/**
+	 * {@return whether this parameter is <a href="https://openjdk.org/jeps/456">unnamed</a>}
+	 */
+	@DerivedProperty
+	boolean isUnnamed();
```

Comment:
Method parameters can not be unnamed. Is this for Lambdas? Maybe clarify that in the docs?